sample-apps/segmentation/lib/scribbles.py

```diff
@@ -14,113 +14,16 @@
 from monailabel.interfaces.tasks.infer import InferTask, InferType
 from monailabel.scribbles.transforms import (
     AddBackgroundScribblesFromROId,
-    ApplyCRFOptimisationd,
     ApplyGraphCutOptimisationd,
-    ApplyISegGraphCutPostProcd,
-    ApplySimpleCRFOptimisationd,
     MakeISegUnaryd,
     MakeLikelihoodFromScribblesHistogramd,
-    SoftenProbSoftmax,
 )
 from monailabel.utils.others.post import BoundingBoxd, Restored
 
 
-class ScribblesInferTask(InferTask):
+class GenericISegGraphcutModelFree(InferTask):
```

Comment:
lets move this class to https://github.com/Project-MONAI/MONAILabel/tree/main/monailabel/scribbles/
and all other sample apps can just use it.. i believe this is generic enough for all other apps to use?

Reply:
@SachidanandAlle this has been done now. I dont have resources to test deepedit/deepgrow apps as it seems to be running some TTA. If you have those setup, can you just load a sample and click update in scribbles (no need to add any scribbles for this test) to do a sanity check?